### DISCUSSION TITLE
500 error on login

### DISCUSSION THREAD

**Discussion by arjitc** [iebb/F1-Web-Viewer#3] [General]
Hi,
I built it from source (npm install, npm run build and npm start), but when I login, it errors with "500 Error"

and on the console I see

```
> f1-web-viewer@1.8.0 start
> node functions/server.js

Server running on port 3000
FetchError: invalid json response body at https://api.formula1.com/v2/account/subscriber/authenticate/by-password reason: Unexpected token < in JSON at position 0
    at /opt/F1-Web-Viewer2/node_modules/node-fetch/lib/index.js:273:32
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async /opt/F1-Web-Viewer2/functions/server.js:53:20 {
  type: 'invalid-json'
}
```

Any help would be appreciated :)

**Comment by Jimgitsit** [comment]
Same here.

**Comment by iebb** [comment]
does it work now?

**Reply by xantrk** [reply]
Mine still doesn't work.
FetchError: invalid json response body at https://api.formula1.com/v2/account/subscriber/authenticate/by-password reason: Unexpected token < in JSON at position 0
    at C:\Users\furka\f1WebViewer\local\F1-Web-Viewer\node_modules\node-fetch\lib\index.js:273:32
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async C:\Users\furka\f1WebViewer\local\F1-Web-Viewer\functions\server.js:53:20 {
  type: 'invalid-json'
}

**Reply by iebb** [reply]
this should be the reese84 not working, have you updated to the latest version? development is switched to beta branch now

**Comment by arjitc** [comment]
Same error here too, exactly the same error.
